tools/hooks/scary_changes.rb

```diff
@@ -72,6 +72,20 @@ def detect_new_table_or_new_column
     EOS
   end
 
+  def detect_column_rename
+    changes = @all.grep(/^dashboard\/db\/migrate\//)
```

Comment:
Looks like this script only scans dashboard migrations.  Is there an analogous script for pegasus migrations?  (If not, no need to add it as part of this change.  Just wanted to make sure there wasn't a second place where we needed to make similar changes.)

Reply:
It doesn't look like we have similar warnings for pegasus DB changes, but we could add it here and have this detect column renames for dashboard or pegasus